README.md

```diff
@@ -98,26 +98,18 @@ and GoTestWAF evaluating ModSecurity. There are two options to run the demo envi
 
 ### Running the demo using Docker
 
-1. Create the Docker network to link GoTestWAF and ModSecurity to. For example, to create the Docker network named `gotestwaf-modsecurity`:
+1. Clone this repository, go to the local folder, run the following commands to create a ModSecurity-based environment:
 
     ```bash
-    docker network create gotestwaf-modsecurity
-    ```
-2. Start containerized [ModSecurity](https://hub.docker.com/r/owasp/modsecurity-crs/) with minimal configuration:
-
-    ```bash
-    docker run -p <PORT_FOR_MODSECURITY>:80 -d -e PARANOIA=1 --rm --network=gotestwaf-modsecurity \
-        owasp/modsecurity-crs:nginx
+    git clone https://github.com/wallarm/gotestwaf.git
+    cd gotestwaf
+    make modsec
     ```
-
-    You will find more ModSecurity configuration options and other image tags on [Docker Hub](https://hub.docker.com/r/owasp/modsecurity-crs/).
-
-    Other options for the ModSecurity launch are described on the [ModSecurity GitHub](https://github.com/SpiderLabs/ModSecurity).
-3. Start containerized GoTestWAF with minimal configuration:
+2. Start containerized GoTestWAF with minimal configuration:
 
     ```bash
-    docker run -v ${PWD}/reports:/go/src/gotestwaf/reports --network=gotestwaf-modsecurity \
-        wallarm/gotestwaf --url=<MODSECURITY_URL>
+    docker run -v ${PWD}/reports:/go/src/gotestwaf/reports --network="host" \
+        wallarm/gotestwaf --url=http://127.0.0.1:8080
```

Comment:
Should not we enable users to create the network themselves? We use this approach in the API Firewall docs, it was recommended by our DevOps

Reply:
In our case, the flow is simpler, modsec is exposed to the localhost via 127.0.0.1:8080, so gotestwaf container should use the system "host" to get access to the container without any additional steps. So we don't need to create an additional network.